google-cloud-storage/src/test/resources/META-INF/native-image/com/google/cloud/storage/reflect-config.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "name":"org.apache.commons.logging.LogFactory",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "allDeclaredConstructors": true
+  },
+  {
+    "name":"org.apache.commons.logging.impl.Jdk14Logger",
+    "methods":[{"name":"<init>","parameterTypes":["java.lang.String"] }]}
+,
+  {
+    "name":"org.apache.commons.logging.impl.LogFactoryImpl",
+    "allDeclaredFields":true,
+    "allDeclaredMethods":true,
+    "methods":[{"name":"<init>","parameterTypes":[] }]}
+,
+{
+  "name":"com.google.cloud.storage.BlobInfo$ImmutableEmptyMap",
```

Comment:
Would you add comment explaining why this configuration is not in main?

Reply:
Sorry, my bad. I unintentionally placed this class config in the test directory. This configuration is used outside of the tests so needs to in the main/resources directory.